docs/content/getting-started/releases.mdx

```diff
@@ -1,28 +1,41 @@
 ---
-title: Releases and Changes | Dagster
+title: Releases and Compatibility | Dagster
 ---
 
-# Releases and Changes
+# Releases and Compability
 
-## Releases
+We follow [Semantic Versioning](https://semver.org/) for compatibility between Dagster releases.
 
-We currently publish two kinds of Dagster releases:
+## Dagster core
 
-- Minor releases, e.g. 0.7.0, 0.8.0. We publish a new minor release every few months.
-- Dot releases, e.g. 0.8.1, 0.8.2. We publish a new dot release at least once per week.
+Dagster's public, stable APIs will not break within any major release. I.e. if a public, stable API exists in Dagster 1.x.y, upgrading to 1.(x+1).y or 1.x.(y+1) should not result in broken code. If a version does break your code, please help us out by filing an issue on [Github](https://github.com/dagster-io/dagster/issues)
 
-## Breaking Changes & Deprecations
+Our public, stable Python API includes:
 
-We are in the 0.y.z of [SemVer](https://semver.org/). While technically this is "anything goes", we provide guidance to set expectations around deprecations and breaking changes:
+- All classes, functions, and variables that are exported at the top-level of the `dagster` package, unless they're marked experimental (more on this [below](#experimental-apis)).
```

Comment:
```suggestion
- All classes, functions, and variables that are exported at the top-level of the `dagster` package, unless they're marked experimental. Refer to the [Experimental APIs section](#experimental-apis) for more info.
```

The parentheses felt a bit awkward to me - suggesting this as a more direct replacement.